docs/wg-terminology.md

```diff
@@ -0,0 +1,39 @@
+# Working Group Terminology
+
+Clarifications and additions.
+
+### RDF Triple
+[RDF Triples are defined in RDF 1.2](https://www.w3.org/TR/rdf12-concepts/#section-triples).
+
+Sometimes informally called: "Triple Type", "Abstract Triple".
+
+### Asserted Triple
+
+A triple is asserted with respect to a graph, when it is an element of a graph.
+
+### RDF Statement
+
+[RDF Statements are defined in RDF 1.2](https://www.w3.org/TR/rdf12-concepts/#dfn-rdf-statement).
+
+(See also [N-quads statement](https://www.w3.org/TR/rdf12-n-quads/#grammar-production-statement).)
+
+The statement is the meaning of the triple.
+
+The triple is the representation of the statement.
+
+### Triple Occurrence
```

Comment:
Perhaps we should continue to use reification terminology and call this a Triple Token.

Reply:
This is the key piece of terminology. It is not going to be easy to find the right word(s).

A triple is a thing in the RDF data model. We have been saying it is a "type" in the sense of a [abstract descriptive concept](https://en.wikipedia.org/wiki/Type%E2%80%93token_distinction). Is "triple" a class? It hasn't been described like that in RDF to-date. https://www.w3.org/TR/rdf12-concepts/#section-triples

If occurrence is part of the RDF data model, the "token" quality seems more like a named occurrence. A named occurrence is a resource (or a blank node that is [mapped to resources by "A", the bnode mapping](https://www.w3.org/TR/rdf12-semantics/#simple)). There can be several named occurrences (resources) for one occurrence.

When we can define terminology, we do need to consider if the word is already in use in RDF. ["Instance" has been suggested](https://lists.w3.org/Archives/Public/public-rdf-star-wg/2024Jan/0069.html) but in RDF that brings in "[an instance of a class](https://www.w3.org/TR/rdf12-schema/#ch_classes)".

Currently, I prefer a name that is not taken from this abstract theoretical framework; rather one that is closer to how it it is used in RDF. Words like "occurrence", "usage", "mention" - all of which bring their own implication-baggage, including they should be "possible occurrence" etc because of itself, it doesn't assume the triple is in-use/asserted anywhere. I doubt we'll find a perfect word.

Reply:
re: "instance", also consider "instance data"